lib/serverengine/server.rb

```diff
@@ -66,12 +66,14 @@ def install_signal_handlers
       s = self
       SignalThread.new do |st|
         st.trap(Daemon::Signals::GRACEFUL_STOP) { s.stop(true) }
-        st.trap(Daemon::Signals::IMMEDIATE_STOP) { s.stop(false) }
-        st.trap(Daemon::Signals::GRACEFUL_RESTART) { s.restart(true) }
-        st.trap(Daemon::Signals::IMMEDIATE_RESTART) { s.restart(false) }
-        st.trap(Daemon::Signals::RELOAD) { s.reload }
         st.trap(Daemon::Signals::DETACH) { s.stop(true) }
-        st.trap(Daemon::Signals::DUMP) { Sigdump.dump }
+        unless ServerEngine.windows?
```

Comment:
ok, then let's add comments here:

```
# Here disables signals excepting GRACEFUL_STOP == :SIGTERM because
# only SIGTERM is available on Windows.
```